src/test/proxy/server_proxy.py

```diff
@@ -0,0 +1,44 @@
+from ja.user.proxy import IUserServerProxy
+from ja.server.database.database import ServerDatabase
+from ja.common.message.server import ServerResponse
+from ja.common.job import Job, JobStatus
+
+from ja.user.config.add import AddCommandConfig
+from ja.user.config.cancel import CancelCommandConfig
+from ja.user.config.query import QueryCommandConfig
+
+
+class ServerProxyDummy(IUserServerProxy):
+
+    def __init__(self, db: ServerDatabase)
+        self._db = db
+
+    def add_job(self, add_config: AddCommandConfig) -> ServerResponse:
+        job: Job = add_config.job
+        db_job: Job = self._db.find_job_by_id(job.uid)
+        if db_job is not None:
+            return ServerResponse(result_string = "Job with id %s already exists" % ,
```

Comment:
I think this error message is broken because you have nothing after the percent sign.